bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -738,6 +627,157 @@ public Image(Device device, ImageDataProvider imageDataProvider) {
 	init();
 }
 
+
+/**
+ * This method copies the image data and add disabled or grayed image depending on the flag
+ * @param data the image data from which original data is copied
+ *
+ * @return new image data with valid styling properties as per the flag
+ */
```

Comment:
Please either revise the comment again, as it is not fitting after the recent API changes anymore, or simply remove it (as the method is private and the name+implementation is easy to comprehend anyway).
- "the flag" is unspecified, as the method does not except a flag. That would have to be "this image's flags" or the like
- data for disabled/grayed is not added but just overwrites the "normal" image data, doesn't it? So the statement about that the method "copies the image data" is not corret, as it is either copied and overwrites it or simply returns the original one (without performing a copy).

I would be in favor of simply having this documentation removed.

Reply:
I will remove it then